node-api-postgres/queries.js

```diff
@@ -48,28 +45,30 @@ const authenticateUser = (request, response) => {
 const getUsers = (request, response) => {
     pool.query('SELECT * FROM users ORDER BY user_id ASC', (error, results) => {
         if (error) {
-            return response.status(400).json(results);
+            return response.status(400).json({});
         }
       
-        response.status(200).json(results.rows);
+        return response.status(200).json(results.rows);
```

Comment:
Correct me if i'm wrong but is there any security issues here? It seems like you're returning all the users in the database? or is it fine in this context?

Reply:
yeah I am returning all users cause the endpoint is for retrieving all users (in case we need it later at some point in time)

right now it is not being called from frontend at all - so in this context it should be ok

although you are right - we need to add some sort of auth check in the api layer to make sure that if these endpoints are manually hit using `curl` or something of the sort, that they are not responding when an unauthed user tries to `getuserbyemail` for example

(I think it is better suited for its own PR - rather than a blocker for this one, what do you think @AdCLeung ?)

Reply:
Cool! maybe add a comment or have a note somewhere so we don't forget to look at it later?

Reply:
yep good call - i will add a ticket in the backlog for our project and also add a todo comment in the code